locales/base/translation.json

```
@@ -777,6 +777,7 @@
   },
   "importExistingKey": {
     "explanation": "Your Recovery Phrase is a 24-word phrase that you wrote down and saved when you set up your account. Please enter it here to restore your account.",
+    "explanation1_50": "Your Recovery Phrase is a 12 or 24 word phrase that you wrote down and saved when you set up your account. Please enter it here to restore your account.",
```

nit: seems we've been using this convention so far ([example](https://github.com/valora-inc/wallet/blob/e822c0465273b4e0784d41cfb3fe01262786f231/locales/base/translation.json#L184))
```suggestion
    "explanationV1_50": "Your Recovery Phrase is a 12 or 24 word phrase that you wrote down and saved when you set up your account. Please enter it here to restore your account.",
```

---

OMG that's what i thought i typed, i even looked for the convention :( idk how the V got lost, thanks!